cni/network/network_windows.go

```diff
@@ -111,7 +111,7 @@ func updateSubnetPrefix(cnsNwConfig *cns.GetNetworkContainerResponse, subnetPref
 
 func getNetworkName(podName, podNs, ifName string, nwCfg *cni.NetworkConfig) (string, error) {
```

Comment:
As discussed, if we don't handle an operation on a given platform we should explicitly throw an error, operation is not support rather silently not doing anything.